framework/include/updater.hpp

```diff
@@ -4,9 +4,26 @@
 
 namespace gvk
 {
-	class updater;
+	class updater;	
 	using event_t = std::variant<std::shared_ptr<event>, files_changed_event, swapchain_changed_event, swapchain_resized_event>;
-	using updatee_t = std::variant<avk::graphics_pipeline, avk::compute_pipeline, avk::ray_tracing_pipeline, avk::image, avk::image_view>;
+	using listener_updatee_t = std::function<void(void)>;
```

Comment:
I find the naming a bit inconsistent: Those callback functions are called "listener" (or "listerer updated") and they are also called "callee" further down). The names should be unified. 

Above, I have expressed the wish to call the method `invoke` but since we already have "invokees" in the framework, we should give them a different name.
- "listener" is okay... 
- but "listener updatee" sounds a bit confusing to me.
- Here is another suggestion: "callback",
- or maybe even better: "update callback"